### PR TITLE
Link to the new maui-compat template

### PR DESCRIPTION
**Pull request by bijington**
## Fixes

Nothing

## Description

Adds a link over to the new MauiCompat template 
- [x] https://github.com/xamarin/XamarinCommunityToolkit/pull/1749

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml
Comment (bijington) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Sadly it seems that just referring to a template file in the url does not apply labels or the title

**Review by brminnick — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml
Comment (bijington) [reply .github/ISSUE_TEMPLATE/config.yml:4]: No problem, I wanted to highlight this as it is now duplicated, the labels are defined in the template file on the XCT repo and also here. I doubt this is likely to change though.

**Comment by brminnick** [comment]
Thanks Shaun!

**Review by brminnick — APPROVED** [review]
I added the full namespace (`Xamarin.CommunitytToolkit.MauiCompat`), just in case someone had never heard of MauiCompat

**Comment by bijington** [comment]
@brminnick sorry we might want to hold off until the merge until the mentioned PR in the description is merged in XCT

**Comment by brminnick** [comment]
No problem!

I got excited after testing [the url](https://github.com/xamarin/XamarinCommunityToolkit/issues/new?assignees=&labels=bug%2C+unverified%2C+maui-compat&template=bug_report_template.md&title=%5BBug%5D+%5BMauiCompat%5D+) and seeing that it loaded the bug report template 😃

**Comment by bijington** [comment]
> No problem!
> 
> I got excited after testing [the url](https://github.com/xamarin/XamarinCommunityToolkit/issues/new?assignees=&labels=bug%2C+unverified%2C+maui-compat&template=bug_report_template.md&title=%5BBug%5D+%5BMauiCompat%5D+) and seeing that it loaded the bug report template 😃 
> 
> 

I was surprised it was working but I've just realised the url wasn't pointing to the new template 🤦

**Comment by bijington** [comment]
@brminnick there is a subtle difference with the URL now to make use of the maui_compat template

**Comment by bijington** [comment]
TODO: Update the link once more if we go ahead with https://github.com/xamarin/XamarinCommunityToolkit/pull/1752

**Review by brminnick — APPROVED** [review]

**Pull request merged**
